sdk/state/ingest.go

```diff
@@ -162,7 +162,8 @@ func (c *Channel) ingestTxMetaToUpdateBalances(resultMetaXDR string) error {
 					continue
 				}
 				ledgerEntryAddress = tl.AccountId.Address()
-				ledgerEntryBalance = int64(tl.Balance)
+				liabilities := tl.Liabilities()
+				ledgerEntryBalance = int64(tl.Balance - liabilities.Selling)
```

Comment:
:bulb: I think it might make sense to rename `ledgerEntryBalance` to `ledgerEntryAvailableBalance` to clarify that this is not just the balance of the ledger entry, but the available balance, the amount that can be spent.

Reply:
I agree
https://github.com/stellar/experimental-payment-channels/pull/311/commits/7f731d3bdd0b1a805cf0888fe9e82f7ab6dd7f72